backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -248,5 +248,19 @@ public static Entity.Project AddTask(this Entity.Project project, params Entity.
             });
             return project;
         }
+
+        /// <summary>
+        /// Throw an exception if the passed property is not in the same agency or sub agency as this project.
+        /// </summary>
+        /// <param name="project"></param>
+        /// <param name="property"></param>
+        /// <returns></returns>
+        public static void ThrowIfPropertyNotInProjectAgency(this Entity.Project project, Entity.Property property)
+        {
+            //properties may be in the same agency or sub-agency of a project. A parcel in a parent agency may not be added to a sub-agency project.
+            if (property.AgencyId != project.AgencyId
```

Comment:
I don't believe this allows a parent agency to sell different sub-agency properties in a single project.

Reply:
I've tested this scenario. Adding a parcel belonging to a sub-agency of the project's agency does not throw this error.